### PR TITLE
improve documentation

### PR DESCRIPTION
**Pull request by lioramilbaum**

### CODE REVIEW

**Comment by lioramilbaum** [comment]
@ynixon @Balex3000 can you look at it and provide feedback?

**Comment by ynixon** [comment]
I could not raise the stage environment using the command `vagrant up stage` 😢 
(`vagrant up dev` works fine)
We should update the documentation telling:
1. Don't to use `vagrant up` command
2. Use only spin.sh/cmd 

We should add  a warning/message to `vagrant up` - to prevent it from running

**Comment by lioramilbaum** [comment]
@ynixon added a comment in the README. Not sure it is worth investing time to prevent using vagrant directly. If you do, let's track it in a different issue.

File: README.md [hunk 1]
```diff
@@ -1,118 +1,43 @@
...
+* AWS account
+* AWS credentials: access key & access secret
+* keypair
+* subnet ID
+* Security Group with inbound ports for SSH (22), HTTP (80) and HTTPS (443)
 
+##### Spin environment
 ```
-vagrant up dev
+./spin.py up [dev|stage]
```
Comment (ynixon) [inline README.md]: Currently does not exists

Comment (lioramilbaum) [reply README.md]: It will be soon enough. I will not merge my PR till it is.

**Pull request merged**